app/services/intl.js

```diff
@@ -4,9 +4,9 @@
  */
 
 import Ember from 'ember';
-import Locale from 'ember-intl/models/locale';
-import createFormatCache from 'ember-intl/format-cache/memoizer';
 import { IntlRelativeFormat, IntlMessageFormat } from 'ember-intl/utils/data';
+import createFormatCache from 'ember-intl/format-cache/memoizer';
```

Comment:
What was the bug? That package has no deps and a single default export. But it's also not compiled to CommonJS, it's simply a pure ES6 module…maybe that's the issue?